src/main/java/org/generationcp/breeding/manager/listmanager/util/GermplasmListExporter.java

```diff
@@ -340,10 +346,34 @@ protected GermplasmList getGermplasmListAndListData(final Integer listId) throws
 
 		final Map<Integer, Variable> variableMap = new HashMap<>();
 		this.addVariableToMap(variableMap, TermId.SEED_AMOUNT_G.getId());
-		this.addVariableToMap(variableMap, TermId.STOCKID.getId());
+		final Variable stockId = createMockStockId();
+		variableMap.put(stockId.getId(), stockId);
 		return variableMap;
 	}
 
+	//	TODO: This method was created to continue export the stockId after removing him FROM cvterm.
+	// 	IBP-4058 Remove StockID variable from the ontology
+	private Variable createMockStockId() {
```

Comment:
Sounds to me that this is not needed, if it is not a variable anymore, then it should not be exported as variable in the file. At the same time, we know that this export is going to die soon. Is it possible to export the column without exporting the variable in the Description sheet? Let's confirm with Mariano tomorrow